docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to form 
+arugments about the proving 
+scheme. 
+
+These arguments are used to 
+check samples of elements, 
+against the original ones. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires and 
+allow us to express circuits
+with less differing vectors 
+per polynomial. 
+
+PLONK uses an argument to check 
+the correctness of a shuffle in 
+its proving algorithm. This allows 
+for the checking of individual 
```

Comment:
"This allows for the checking of individual wires rather than entire sets." Can you explain what you mean here? With the above sentence, it seems to imply that the permutation polynomial allows you to check individual wires rather than sets?